### PR TITLE
Tasks: 1130-1145 X-CSR commonsense multiple QA

### PR DESCRIPTION
**Pull request by Maitreyapatel**
Please review these set of task files having commonsense QA from multiple languages.

### CODE REVIEW

**Comment by danyaljj** [comment]
@Sujan242 wondering if you can review this PR?

**Comment by Sujan242** [comment]
Looks good 
  
1) I think the fields "Input_language" and "output_language" are redundant and are not part of the JSON file template that is used in this repository. So please remove them.

2) Also, ensure that labels are balanced across instances in all the tasks.

**Comment by Maitreyapatel** [comment]
@Sujan242 Thanks for reviewing these tasks. 
1. It's been decided to keep the "Input_language" and "output_language" fields, if and only if we are using difference languages. Please refer to the issue #113 .
2. Yes, I made sure that data is balanced. There are ~200 instances for each options in every task files.

Let me know your thoughts on this!

**Comment by Sujan242** [comment]
> 2. Yes, I made sure that data is balanced. There are ~200 instances for each options in every task files.

In some tasks, the maximum frequency of a label is around 220 and the minimum is around 170. 
@danyaljj is this amount of skewness fine given that there are lesser than 1000 instances. 
Everything else looks good to me.

File: tasks/task1130_xcsr_vi_commonsense_mc_classification.json [hunk 1]
```diff
@@ -0,0 +1,6014 @@
...
+    "Source": [
+        "X-CSR-QA dataset",
+        "Link: https://inklab.usc.edu/XCSR/xcsr_datasets"
+    ],
+    "Categories": [
+        "Multiple-Choice Question"
+    ],
+    "Input_language": "vi",
+    "Output_language": "en",
+    "Definition": "In this task, you will be presented with a multiple-choice question, and you should answer the question based on your commonsense knowledge. Moreover, the question and options are given in Vietnamese language. Classify the answers based on the given options.",
```
Comment (danyaljj) [inline tasks/task1130_xcsr_vi_commonsense_mc_classification.json]: Say what is the expected output (is it the answer string or "A", "B", "C", ...)

Comment (danyaljj) [reply tasks/task1130_xcsr_vi_commonsense_mc_classification.json]: Shorten the instructions. I think you can merge and shorten the first two sentences.

Comment (danyaljj) [reply tasks/task1130_xcsr_vi_commonsense_mc_classification.json]: (obviously, these apply to all tasks)

Comment (Maitreyapatel) [reply tasks/task1130_xcsr_vi_commonsense_mc_classification.json]: I have updated all definitions!!

**Pull request merged**